lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 47px auto;
+    grid-template-rows: 47px auto;
+    gap: 0 16px;
+    grid-template-areas:
+      "logo top-row"
+      "logo bottom-row";
+
+    height: 72px;
+    margin-bottom: 0;
+  }
+
+  .nx-product__logo-image {
+    display: block;
+  }
+
+  .nx-product__logo {
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 1;
+    grid-area: logo;
+  }
+
+  .nx-product__summary {
+    -ms-grid-row-align: start;
+    align-self: end;
```

Comment:
Shouldn't these two match?

Reply:
> Shouldn't these two match?

Yeah though IE is still a bit goofy about the grid, I'm calling IE "good enough" for the one month it will be in place: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/ba288f85a0d98a0dbc937040d86f28226f48b48f